package.json

```diff
@@ -160,7 +160,7 @@
     "nyc": {
         "report-dir": "coverage/cypress/"
     },
-    "packageManager": "pnpm@10.8.0",
+    "packageManager": "pnpm@10.22.0+sha512.bf049efe995b28f527fd2b41ae0474ce29186f7edcb3bf545087bd61fbbebb2bf75362d1307fda09c2d288e1e499787ac12d4fcb617a974718a6051f2eee741c",
     "pnpm": {
         "onlyBuiltDependencies": [
             "cypress",
```

Comment:
can you pin these to versions as well? we get that in 10.19 :)

Reply:
Good idea! Yes, I will!

Reply:
thank you! looks good to me but I'll wait for @kevinb-khan to approve